### PR TITLE
Term entry Javascript storage set item

### PR DESCRIPTION
**Pull request by Manikanta528**
### Description

Addressing issue #2693 adding new term entry to storage/terms/setItem.md and  Adding new tag - Storage to tags.md

### Type of Change

- Adding a new entry Javascript storage set item
- Adding new tag - Storage to tags.md

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/storage/terms/setItem.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+Title: '.setItem()' 
+Description: 'updates the value to the specified storage object key if exists, else adds the key to the specified storage object.'
```
Comment (caupolicandiaz) [inline content/javascript/concepts/storage/terms/setItem.md:3]:
```suggestion
Description: 'Updates the value to the specified storage object key, if it exists, else adds the key to the object.' 
```

File: content/javascript/concepts/storage/terms/setItem.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+---
+Title: '.setItem()'
```
Comment (caupolicandiaz) [inline content/javascript/concepts/storage/terms/setItem.md:2]: looks like this isn't in the correct location. It should be within a folder named "setItem"

File: content/javascript/concepts/storage/terms/setItem.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+Tags: 
+  - 'Storage'
+  - 'Objects'
+  - 'Methods'
+CatalogContent: 
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`.setItem()`** method updates the `value` to the specified storage object `key` if exists, else adds the `key` to the specified storage object and returns `undefined`.
```
Comment (caupolicandiaz) [inline content/javascript/concepts/storage/terms/setItem.md:16]:
```suggestion
The **`.setItem()`** method updates the value to the specified storage object key, if it exists, else adds the key to the specified storage object and returns `undefined`.
```

Comment (caupolicandiaz) [reply content/javascript/concepts/storage/terms/setItem.md:16]: backticks are used for specific terms or elements within a code block or the language generally. However, generic terms such as class or string don't need to be formatted.

File: content/javascript/concepts/storage/terms/setItem.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+```pseudo
+localStorage.setItem(key, value);
+sessionStorage.setItem(key, value);
+```
+
+Both `key` and `value` are required parameters.
+- The `key` is the name of the key to create.
+- The `value` is the value to create/update the `key`.
+
+
```
Comment (caupolicandiaz) [inline content/javascript/concepts/storage/terms/setItem.md:29]: remove empty line. There should never be multiple empty lines between content.

File: content/javascript/concepts/storage/terms/setItem.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+## Example
+
+This is the example, which demonstrates the usage of `.setItem()` method:
```
Comment (caupolicandiaz) [inline content/javascript/concepts/storage/terms/setItem.md:32]:
```suggestion
This example demonstrates a basic implementation of the `.setItem()` method:
```

File: content/javascript/concepts/storage/terms/setItem.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+```js
+console.log(localStorage.setItem('name', 'mark'));
+localStorage.setItem('gender', 'male');
+console.log("Gender :",localStorage.gender);
+```
+
+The Output will be:
+
+```bash
```
Comment (caupolicandiaz) [inline content/javascript/concepts/storage/terms/setItem.md:42]:
```suggestion
```shell
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@Manikanta528 I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

File: content/javascript/concepts/storage/terms/setItem.md (hunk 3 above)
Comment (Manikanta528) [reply content/javascript/concepts/storage/terms/setItem.md]: okay

**Comment by Manikanta528** [comment]
@caupolicandiaz updated the requested changes.

**Review by caupolicandiaz — APPROVED** [review]
@Manikanta528 Looks good. I'm passing this along for a second review.

**Review by SSwiniarski — APPROVED** [review]
@Manikanta528 This looks good to merge once a "Storage" entry is merged.

**Comment by SSwiniarski** [comment]
@Manikanta528 Congrats on the new entry! Live link here: https://www.codecademy.com/resources/docs/javascript/storage/setItem

image

**Pull request merged**